presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -323,3 +323,37 @@ This only applies if ``system-mem-pushback-enabled`` is ``true``.
 
 Specifies the amount of memory to shrink when the memory pushback is
 triggered. This only applies if ``system-mem-pushback-enabled`` is ``true``.
+
+Environment Variables As Values For Worker Properties
+-----------------------------------------------------
+
+This section applies to worker configurations in the config.properies file
+and catalog property files only.
+
+The value in a key-value par may reference an environment variable by using
+a leading `$` followed by enclosing the environment variable name in brackets (`{}`).
+
+```
+key=${ENV_VAR_NAME}
+```
+
+The environment variable name must match exactly with the defined variable.
+
+This allows a worker to read sensitive data e.g. for access keys from an
+environment variable rather than having the actual value hard coded in a configuration
+file on disk enhancing the security of deployments.
+
+For example, consider the hive connector's `hive.s3.aws-access-key` property.
+This is sensitive data and can be stored in an environment variable, for example,
+`AWS_S3_ACCESS_KEY` which is set to the actual access key value.
+
+One mechanism is to create a preload library that is injected at the time
+presto_server is started. It decrypts encrypted secrets and sets environment
+variables specific to the presto_server process. These can then be referenced
+in the properies.
+
+For example, once decrypted the preloaded library sets the AWS_S3_ACCESS_KEY
+environment variable which then can be accessed by providing it in the catlog properties:
```

Comment:
```suggestion
environment variable which then can be accessed by providing it in the catalog properties:

```
Add a line break before the code block for the code block to format properly. This is what it looks like in a local doc build: 
<img width="760" alt="Screenshot 2024-12-02 at 10 10 56 AM" src="https://github.com/user-attachments/assets/510de63c-8a1b-45a7-9b35-ae562f5dd9c5">

Also, same comment as in line 366 asking if ` and space before and after is what you want displayed in the code block.